crates/pathfinder/src/rpc/api.rs

```diff
@@ -392,7 +403,12 @@ impl RpcApi {
 
     /// Return the currently configured StarkNet chain id.
     pub async fn chain_id(&self) -> RpcResult<StarknetChainId> {
```

Comment:
Should this not just be `RpcResult<String>` -- then we don't need any of this boilerplate?